src/dx/pubnub_client.rs

```diff
@@ -131,8 +131,8 @@ where
     pub(crate) transport: T,
 
     /// Instance ID
-    #[builder(setter(strip_option), default = "None")]
-    pub(crate) instance_id: Option<String>,
+    #[builder(setter(strip_option), default = "Arc::new(Some(\"None\".to_string()))")]
+    pub(crate) instance_id: Arc<Option<String>>,
```

Comment:
Does it mean that user has to provide the the ready to use Arc with the instance id?

I think that it will be better to add user ability to pass the String to the setter and wrap it to the arc on our own. 

Here [reference](https://docs.rs/derive_builder/latest/derive_builder/#completely-custom-fields-in-the-builder).

Reply:
`Some` of `None`? Does rust checker say something about this default: `Arc::new(None)`?

Reply:
`Arc::new(None)` is valid

Reply:
I think now it will be convenient for user to provide value without worrying about Arc